### DISCUSSION TITLE
Validation and Errors

### DISCUSSION THREAD

**Discussion by AhmedSuror** [techlab/jquery-smartwizard#138] [Q&A] [answered]
I cannot figure out the best approach for validation, also, I Implemented my manual validation but when using errorSteps it doesn't work!
`    $('#smartwizard').smartWizard({
      errorSteps: [1] // Highlight step with errors
    });`
Also, how to show finish button at the last step?

**Comment by techlab** [comment, marked as answer]
Please check here validation example for v6: https://techlaboratory.net/projects/demo/jquery-smart-wizard/v6/validation#step-1
Code example: https://github.com/techlab/jquery-smartwizard/blob/master/examples/validation.html

The example shows how the "Confirm Order" button is enabled on only the last step, you can use the same approach to show/hide also.
